addons/html_editor/static/tests/html_field.test.js

```diff
@@ -992,8 +992,8 @@ test("'Video' command is not available by default when sanitize = true", async (
     setSelectionInHtmlField();
     insertText(htmlEditor, "/video");
     await animationFrame();
-    expect(".o-we-powerbox").toHaveCount(0);
-    expect(queryAllTexts(".o-we-command-name")).toEqual([]);
+    expect(".o-we-powerbox").toHaveCount(1);
+    expect(queryAllTexts(".o-we-command-name")).toEqual(["Video Link"]);
```

Comment:
Who is addind "Video Link" ?

Reply:
Me, to be debated, but I think we need a flag for "disableVideo" because we CAN'T have them, and another flag for "disableVideo" because we DON'T WANT them

To me, the current `disableVideo` feels like a CAN'T (will discuss this with POs)